Packs/UpdateObject/pack_metadata.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "UpdateObject",
+    "description": "Updates the values within an array of objects for a given key.\n\nExample Command:\n!UpdateObject original_object=${myobject} key_to_update=\"Key1\" values=\"newval1,newval2,newval3\"\n\nResults:\n\nOriginal Object (myobject):\n[\n    {\n        'Key1': 'oldval1',\n        'Key2': 'key2val1'\n    }, {\n        'Key1': 'oldval2',\n        'Key2': 'key2val2'\n    }, {\n        'Key1': 'oldval3',\n        'Key2': 'key2val3'\n    }\n]\n\nUpdated Object (mynewobject):\n[\n    {\n        'Key1': 'newval1',\n        'Key2': 'key2val1'\n    }, {\n        'Key1': 'newval2',\n        'Key2': 'key2val2'\n    }, {\n        'Key1': 'newval3',\n        'Key2': 'key2val3'\n    }\n]",
```

Comment:
The pack description should not consist of examples.

Reply:
Addressed in https://github.com/xsoar-contrib/content/pull/2